content/millennium/r4/general-clinical/condition.md

```diff
@@ -25,6 +25,7 @@ The following fields are returned if valued:
 * [Resolved date  (only applies to problems)](https://hl7.org/fhir/R4/condition-definitions.html#Condition.abatement_x_)
```

Comment:
Update as 'Resolved date (only applies to problems and health concerns)' and remove additional space between date and (

Reply:
Addressed